programs/summerofcode/2025.md

```diff
@@ -31,3 +31,17 @@ You can find the project ideas from previous year [here](./2024.md).
 
 ## Ideas
 
+#### ORAS
+
+##### Enhance Java ORAS SDK
+
+- Description: The ORAS project aims to enhance its Java SDK to support a broader range of features from the OCI Distribution spec. This involves implementing missing functionality, improving existing features, and expanding the SDK’s overall capabilities.
+- Expected Outcome:
+  - Implement missing features from the OCI Distribution and Image Specifications, such as [chunked uploads](https://github.com/opencontainers/distribution-spec/blob/main/spec.md#pushing-blobs) and the [Referrers API](https://github.com/opencontainers/distribution-spec/blob/main/spec.md#endpoints)
+  - Improve existing features, robustness and tests to ensure full compatibility with the OCI Distribution and Image Specifications.
+  - Enhance documentation and provide more comprehensive examples.
+  - Add support for additional authentication methods, including using credentials from docker config.json
+- Recommended Skills: java, oci
+- Mentor(s):
+  - Valentin Delaye (@jonesbusy, jonesbusy@gmail.com) - primary
+- Upstream Issues: https://github.com/oras-project/oras-java/issues
```

Comment:
I recommend creating an upstream issue (or an umbrella issue if there are several related issues covered in the project). This will give prospective candidates a place to look for more details, and a central spot for them to ask questions while they develop their proposals.